jsonnet/kube-prometheus/addons/podsecuritypolicies.libsonnet

```diff
@@ -87,6 +87,15 @@ local restrictedPodSecurityPolicy = {
         namespace: $.values.alertmanager.namespace,
       }],
     },
+
+    podSecurityPolicy: restrictedPodSecurityPolicy {
+      metadata+: {
+        name: 'alertmanager-psp',
+      },
+      spec+: {
+        volumes+: ['projected'],
```

Comment:
Why not just add it into `restrictedPodSecurityPolicy.spec.volumes` (L15-L21)?

Reply:
It would give permission to other components that don't use projected volumes, but yeah... I think that is not a big deal.

I'll change this again